### ISSUE TITLE
version script usage with extensions

### ISSUE THREAD

**Issue by Ashton-W** [thoughtbot/liftoff#222]
The version script does not play nicely with iOS extensions, specifically WatchKit App. It complains that the version numbers are different. I tried applying `liftoff` to my extension targets but that didn't change it.

Also the 'WatchKit App' extension doesn't have a *Build Phases* section in Xcode - but liftoff is adding the scripts. This is probably an **xcodeproj** issue.

**Comment by gfontenot** [comment]
Yeah, I saw that was going to be an issue. Haven't had a chance to figure out a solution. Bummer.
